mithril-signer/src/runtime/runner.rs

```diff
@@ -514,6 +533,13 @@ mod tests {
         ));
         let kes_signer = None;
 
+        let network_configuration_service = Arc::new(FakeMithrilNetworkConfigurationProvider::new(
+            Default::default(),
+            Default::default(),
+            Default::default(),
+            ticker_service.clone(),
+        ));
```

Comment:
The instantiation of the Fake provider is hard to read with all these `Default::default()`, can you try to make it more readable?